### PR TITLE
[master] Porting #49482 to master

### PR DESCRIPTION
**Pull request by garethgreenaway**
Porting #49482 to master

### CODE REVIEW

File: salt/utils/network.py [hunk 1]
```diff
@@ -1896,8 +1896,16 @@ def refresh_dns():
         pass
 
 
+@jinja_filter('connection_check')
+def connection_check(addr, port=80, safe=False, ipv6=None):
```
Comment (dwoz) [inline salt/utils/network.py]: I don't see the need for `connection_check` when `dns_check` does the *exact* same thing.

File: salt/utils/network.py [hunk 2]
```diff
@@ -1896,8 +1896,16 @@ def refresh_dns():
...
+    '''
+    Provides a convenient alias for the dns_check filter.
+    '''
+    return dns_check(addr, port, safe, ipv6)
+
+
 @jinja_filter('dns_check')
-def dns_check(addr, port, safe=False, ipv6=None):
+def dns_check(addr, port=80, safe=False, ipv6=None):
```
Comment (dwoz) [inline salt/utils/network.py]: Should this default to `None` instead of 80?

Also, this change should be tested.

**Review by dwoz — CHANGES_REQUESTED** [review]

File: salt/utils/network.py (hunk 2 above)
Comment (garethgreenaway) [reply salt/utils/network.py]: Added a test case to the dictionary that does not include the port.  Updated the call to dns_check to not pass port if it's not in the dictionary.  Updated the default to be None.

File: salt/utils/network.py (hunk 1 above)
Comment (garethgreenaway) [reply salt/utils/network.py]: @MTecknology Looks like you originally added the `connection_check` alias, would you care to comment on it's need versus `dns_check`?

Comment (MTecknology) [reply salt/utils/network.py]: @garethgreenaway I don't recall adding it and I'm having a hard time finding a commit where I added it. Is there any chance you could point me toward it so maybe my memory can be triggered?

Comment (twangboy) [reply salt/utils/network.py]: https://github.com/saltstack/salt/pull/49482/commits/026a5763e31731cfb66509f3ac7eea20f87d078a

Comment (twangboy) [reply salt/utils/network.py]: @MTecknology ^^^

Comment (MTecknology) [reply salt/utils/network.py]: Oh, hey friends! How have you fellas been?

Yes, now that I'm digging back into things, I do recall this; ``connection_check()`` was a "convenience alias" in [#46053](https://github.com/saltstack/salt/issues/46053#issuecomment-366277516).

If we prefer not have aliases, then I see no reason to keep it.

Comment (MTecknology) [reply salt/utils/network.py]: Removing this function would effectively finish reverting commits 2e0ae24 and 026a576. I believe this is a good idea.

**Review by MTecknology — APPROVED** [review]

**Comment by dwoz** [comment]
Closing this due to inactivity.  Anyone should feel free to re-open it if they want to see it through to the end in one release cycle.

**Comment by MTecknology** [comment]
So, was this abandoned due to staff turnover, or what happened?

**Comment by twangboy** [comment]
Pretty much

**Pull request closed without merging**